app/assets/stylesheets/components/tag.scss

```diff
@@ -19,6 +19,34 @@
   &--inactive {
     background-color: govuk-colour("dark-grey");
   }
+
+  &--light-green {
+    color: govuk-shade(govuk-colour("green"), 20);
+    background: govuk-tint(govuk-colour("green"), 80);
+  }
+
+  &--light-grey {
+    color: govuk-shade(govuk-colour("dark-grey", $legacy: "grey-1"), 30);
+    background: govuk-tint(govuk-colour("dark-grey", $legacy: "grey-1"), 90);
+  }
+
+  &--light-red {
+    color: govuk-shade(govuk-colour("red"), 30);
+    background: govuk-tint(govuk-colour("red"), 80);
+  }
+
+  &--light-yellow {
+    color: govuk-shade(govuk-colour("yellow"), 65);
+    background: govuk-tint(govuk-colour("yellow"), 75);
+  }
+
+  &--dark-green {
+    background-color: govuk-colour("green");
+  }
+
+  &--dark-red {
+    background-color: govuk-colour("red");
+  }
```

Comment:
Unfortunately the SCSS is already a bit of a mess. We have to redefine the styles here because of the `.govuk-tag` rule in `gds_kit_tweaks.scss`. I've changed the style names to be more similar so refactoring's easier.

Not sure what your other comments are referring to. I'm of course aware of and have read the docs. The styles used are a combination of the multiple status tags and the styles we already have. There's nothing new here apart from we're now using multiple statuses.

Reply:
@nomasprime are you saying we cannot use then `govuk-tag--green` for example?
I think this is what you are saying